tests/flash.c

```diff
@@ -93,6 +95,51 @@ static struct Test tests[] = {
           .size = 0,
           .reset = 1,
           .log_level = DEBUG_LOG_LEVEL,
+          .freq = 0,
+          .format = FLASH_FORMAT_BINARY }
+    },
+    { "--debug --freq 5k --reset write test.bin 0x80000000", 0,
+        { .cmd = FLASH_CMD_WRITE,
+          .serial = { 0 },
+          .filename = "test.bin",
+          .addr = 0x80000000,
+          .size = 0,
+          .reset = 1,
+          .log_level = DEBUG_LOG_LEVEL,
+          .freq = 5,
+          .format = FLASH_FORMAT_BINARY }
+    },
+    { "--debug --freq 15K --reset write test.bin 0x80000000", 0,
+        { .cmd = FLASH_CMD_WRITE,
+          .serial = { 0 },
+          .filename = "test.bin",
+          .addr = 0x80000000,
+          .size = 0,
+          .reset = 1,
+          .log_level = DEBUG_LOG_LEVEL,
+          .freq = 15,
+          .format = FLASH_FORMAT_BINARY }
+    },
+    { "--debug --freq=5k --reset write test.bin 0x80000000", 0,
+        { .cmd = FLASH_CMD_WRITE,
+          .serial = { 0 },
+          .filename = "test.bin",
+          .addr = 0x80000000,
+          .size = 0,
+          .reset = 1,
+          .log_level = DEBUG_LOG_LEVEL,
+          .freq = 5,
+          .format = FLASH_FORMAT_BINARY }
+    },
```

Comment:
Duplicate?

Reply:
The one is in another syntax (--freq=5k), the former one is "--freq 5k"

Reply:
Correct, I recognise the slight difference now.